torch/_torch_docs.py

```diff
@@ -1242,6 +1242,46 @@ def merge_dicts(*dicts):
     tensor([ True, False, False])
 """.format(**common_args))
 
+add_docstr(torch.bitwise_left_shift,
+           r"""
+bitwise_left_shift(input, other, *, out=None) -> Tensor
+
+Computes the left arithmetic shift of :attr:`input` by :attr:`other` bits.
+Both :attr:`input` and :attr:`other` must have integer dtypes. The result will have the same dtype as :attr:`input`.
```

Comment:
This will no longer be true once the current left shift implementation is used. What actually happens with floating point inputs? We should probably document the behavior using cases ("If input has an integer dtype... If input has a floating point dtype..."). We should be sure to document the odd behavior that int tensor x floating point scalar inputs have their floating point scalar values cast to the tensor's dtype.